src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaVariableDefinition.java

```diff
@@ -21,7 +21,8 @@
     TASK_TYPE("taskType"),
     TITLE("title"),
     HAS_WARNINGS("hasWarnings"),
-    WARNING_LIST("warningList");
+    WARNING_LIST("warningList"),
+    CASE_MANAGEMENT_CATEGORY("caseManagementCategory");
 
```

Comment:
If the new variable is one of the required variables then we also need to update the tests and the configuration api logic to add it as a local task variable

Reply:
I had more code in the new commit